### DISCUSSION TITLE
neovim 0.10 is released

### DISCUSSION THREAD

**Discussion by chipsenkbeil** [nvim-orgmode/orgmode#726] [General]
https://github.com/neovim/neovim/releases/tag/v0.10.0

@kristijanhusak I think you mentioned somewhere the intention to drop 0.9 fairly quickly after 0.10 got released. I want to make sure that org-roam.nvim follows you.

Do you have any idea on a timeline for when you'd have nvim-orgmode stop supporting 0.9.x? I've seen some configurations like the virtual indent that only work for 0.10, so curious on your plans.

**Comment by kristijanhusak** [comment]
I'm trying to keep at least last 3 versions. Considering those are 0.9.4, 0.9.5 and now 0.10, i don't think I'll do anything yet. Once we get some patches for 0.10  I'll slowly start dropping <0.10 support.
